source/ranges.tex

```diff
@@ -6313,13 +6332,13 @@
     constexpr auto begin() requires (!@\exposconcept{simple-view}@<V>)
     { return @\exposid{iterator}@<false>(ranges::begin(@\exposid{base_}@)); }
 
-    constexpr auto begin() const requires @\exposconcept{simple-view}@<V>
+    constexpr auto begin() const requires range<const V>
```

Comment:
Why is this not `\libconcept{range}<const V>`? I don't understand when we do use `\libconcept` and when we don't.

Reply:
Nor do I, but I note that the occurrences of `common_range` on 6338 and 6341 are also not `\libconcept`ed.

Reply:
The convention was to not use libconcept in synopsis/itemdecls (don't know why, maybe to reduce the visual noise or for copy-paste reasons?), so I followed that. Since then, Jens made a big change to use them inside synopsis/itemdecls also, to improve indexing. So now, any libconcepts should be \libconcept'ed. 
 That said, there are still quite a few cases in ranges.tex that don't follow this, and in this case I was following the local convention which didn't. They should all be fixed. I'll go ahead and fix this one since you've pointed it out (and it's new wording). The rest should be fixed as well - I opened #4383 for them.

Reply:
I find it makes the LaTeX much less readable to have `\libconcept` on every concept, and so harder to review.